tests/test_config.cpp

```diff
@@ -1,20 +1,206 @@
 #include <GameConfig.hpp>
+
+#include <boost/filesystem.hpp>
 #include <boost/test/unit_test.hpp>
+
 #include <fstream>
+#include <map>
+
+namespace fs = boost::filesystem;
+
+typedef std::map<std::string, std::map<std::string, std::string>> simpleConfig_t;
+
+simpleConfig_t getValidConfig() {
+    simpleConfig_t result;
+    // Don't change game.path and input.invert_y keys. Tests depend on them.
+    result["game"]["path"] = "\t/dev/test  \t \r\n";
+    result["game"]["\tlanguage\t "] = "      american ;american english french german italian spanish.";
```

Comment:
nit: maybe add a comment to make clear this is intended and maybe add some reasoning.